### PR TITLE
Add submission confirmation page

### PR DESCRIPTION
**Pull request by Jkrzy**
[Link to ZenHub issue.](https://app.zenhub.com/workspaces/doj-crt-intake-scrum-board-5d03bf56c1c8a35d482eca0f/issues/18f/crt-portal/79)

## What does this change?

Updating last step of report submission workflow to display reference ID for the submitted report, content about what to expect next, as well as complete details of the report which was just submitted.
 
On the backend, this includes some work to move our template rendering towards leveraging the report model as opposed to building dictionaries to be passed into the context. My hope is this will ease future changes and give us a bit more flexibility.

I've intentionally contained the new .scss code to a new file to simplify review and ease refactoring as I suspect this initial implementation will need revision.

## Screenshots (for front-end PR):
page8-confirmation

### Author

+ [x] If this is a story, run locally and check to make sure all Acceptance Criteria are met. (If any criteria are unclear, ask about them.)
+ [x] Check for [accessibility](/docs/a11y_plan.md).
+ [x] [Tests pass](https://github.com/USDOJ/crt-portal/#tests).

### Reviewer

+ [ ] If this is a story, run locally and check to make sure all Acceptance Criteria are met. (If any criteria are unclear, ask about them.)
+ [ ] [Tests pass](https://github.com/USDOJ/crt-portal/#tests).

## Notes for reviewer:

See [PR instructions doc](https://github.com/usdoj/crt-portal/blob/master/docs/pull_requests.md) for full pull request review instructions.

### CODE REVIEW

**Comment by Jkrzy** [comment]
@LindsayYoung  -- There's a handful of remaining style tweaks to be made here but I think it's ready for an initial look at the code.

I went ahead and targeted the `pro-form` branch as I suspect work there will complete before this branch is ready to merge.

File: pa11y_scripts/.pa11y-ci-p8-confirmation.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
+{
```
Comment (LindsayYoung) [inline pa11y_scripts/.pa11y-ci-p8-confirmation.json]: This is great, we should also add it to the checks we run at build time
https://github.com/usdoj-crt/crt-portal/blob/develop/.circleci/config.yml#L63

**Comment by LindsayYoung** [comment]
This looks good overall! One comment on adding the new A11y test to CI. 

I noticed an issue with the Date page title showing up twice, but I think that will be resolved in the proform pr. We can circle back on that when the other PR is done.

style stuff:
(documenting what I noticed for thoroughness, but it's is out of scope for this review since you are still working)
 - record number is smaller
 - paper icon 
- What to expect needs more blue above it

**Comment by Jkrzy** [comment]
Fixes in for icon and `h2` top padding. I don't think I'm seeing the issue with `record number`, let's chat about it today.

**Comment by Jkrzy** [comment]
Going to rebase this against `pro-form`

**Pull request merged**